docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -95,6 +95,10 @@ If you must request a UserInfo JSON object from the OIDC `UserInfo` endpoint, se
 A request is sent to the OIDC provider `UserInfo` endpoint, and an `io.quarkus.oidc.UserInfo` (a simple `javax.json.JsonObject` wrapper) object is created.
 `io.quarkus.oidc.UserInfo` can be injected or accessed as a `SecurityIdentity` `userinfo` attribute.
 
+`quarkus.oidc.authentication.user-info-required` is enabled automatically if `quarkus.oidc.roles.source` is set to `userinfo` or `quarkus.oidc.token.verify-access-token-with-user-info` is set to `true` or `quarkus.oidc.authentication.id-token-required` is set to `false`, the current OIDC tenant must support a UserInfo endpoint in these cases.
+
+`quarkus.oidc.authentication.user-info-required` is also enabled automatically if `io.quarkus.oidc.UserInfo` injection point is detected but only if the current OIDC tenant supports a UserInfo endpoint.
```

Comment:
```suggestion
`quarkus.oidc.authentication.user-info-required` is automatically enabled if one of these conditions is met: 

- if `quarkus.oidc.roles.source` is set to `userinfo` or `quarkus.oidc.token.verify-access-token-with-user-info` is set to `true` or `quarkus.oidc.authentication.id-token-required` is set to `false`, the current OIDC tenant must support a UserInfo endpoint in these cases.

- if `io.quarkus.oidc.UserInfo` injection point is detected but only if the current OIDC tenant supports a UserInfo endpoint.
```

Reply:
Thanks @gastaldi, it should read better in the docs, will do the same for another doc